OpenLiberty/guide-cloud-ibm
# Merge qa to master: GH Actions - Daily Builds (#101)

* Update scripts

* Updates to test scripts

* Script fix

* add sudo

* Move Minikube setup steps

* Update to include testing daily OL docker image

* Move Minikube start script

* Add helm uninstall

* Change permissions on running Minikube setup

* temp add pwd command

* add another pwd to check for directory changes

* Remove pwd, make testApp.sh executable

* Add delay between tests

* clean package and build

* Add sudo

* Clear .m2 cache

* Force kill processes

* Remove process kills

* Use script to stop and delete minikube between tests

* Add missing start minikube

* Check docker image build level

* Move minikube commands to execute in same shell, move docker image tests to their own file

* Force link

* Update test.yml

* Update test.yml

Co-authored-by: Gilbert Kwan <gkwan@ca.ibm.com>
Co-authored-by: Austin Seto <austin.seto@ibm.com>
